etl/steps/data/garden/nasa/2023-03-06/ozone_hole_area.meta.yml

```diff
@@ -0,0 +1,57 @@
+all_sources:
+  - source_testing: &source-testing
+      name: NASA
+      published_by: NASA Ozone Watch
+      url: http://ozonewatch.gsfc.nasa.gov/meteorology/annual_data.html
+      date_accessed: 2023-03-06
+      publication_date: 2023-02-14
+      publication_year: 2023
+      # description: Source description.
+
+dataset:
+  title: Ozone hole area and concentration (NASA, 2023)
+  description: |
+    Annual maximum and Antarctic stratospheric ozone hole area, resultant from the emission of ozone-depleting substances.
```

Comment:
Shouldn't it be "resulting from" instead of "resultant from"?

Reply:
Good point. I'll leave it for now, as this is what the author wrote in the previous dataset (https://owid.cloud/admin/datasets/2723)